types/serverless/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for serverless 1.18
+// Project: https://github.com/serverless/serverless#readme
+// Definitions by: Sebastian Müller <https://github.com/sbstjn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Serverless {
```

Comment:
These dont look correct since https://github.com/serverless/serverless/blob/master/lib/Serverless.js#L18 shows that serverless is a class with methods, init, run, setProvider, getProvider and such method, which seems to be missing,